apispec/ext/marshmallow/swagger.py

```diff
@@ -250,7 +250,7 @@ def field2property(field, spec=None, use_refs=True, dump=True, name=None):
         elif spec:
             ret.update(resolve_schema_dict(spec, field.schema))
```

Comment:
Does this line need `dump=dump` as well?  My test passes without so I didn't add this.